app/snapshot.py

```diff
@@ -31,10 +32,7 @@ def to_header() -> [str]:
                 'Monthly Price',
```

Comment:
I found that the spreadsheet doesn't sort the price columns for instances correctly because they contain a literal dollar sign.

![image](https://user-images.githubusercontent.com/5783817/204622450-e580d900-0c3d-46b2-b593-a80c68034d11.png)

I think the contents of the cell needs to be without the dollar sign and then format as "Currency". Then it will appear with the dollar sign, but Excel will do calculations without the dollar sign.

This only seems to apply to instances, the other types don't have the dollar sign.

Reply:
I've noticed the same thing, it must be due to the way instance prices are calculated. I'll get them all uniform!